### PR TITLE
(maint) Amazon Updates

### PR DESCRIPTION
**Pull request by doug-rosser**
These updates allow simulations in Amazon ec2.

### CODE REVIEW

File: simulation-runner/config/scenarios/OpsWorksLrgPGSync.json [hunk 1]
```diff
@@ -0,0 +1,12 @@
+{
+    "run_description": "role::by_size::medium, 240 agents, 480 second rampup, 0-240 second splay, once",
```
Comment (haus) [inline simulation-runner/config/scenarios/OpsWorksLrgPGSync.json]: I assume splay was manually configured in PE for this test case?

File: simulation-runner/src/main/scala/com/puppetlabs/gatling/node_simulations/OpsWorksLarge.scala [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+// import io.gatling.jdbc.Predef._
+
+class OpsWorksLarge extends SimulationWithScenario {
+
+// 	val httpProtocol = http
+// 		.baseURL("https://aws-lrg-mom.us-west-2.compute.internal:8140")
+
+	val reportBody = ElFileBody("OpsWorksLarge_0005_request.txt")
+
+	val headers_0 = Map("X-Puppet-Version" -> "4.10.4")
```
Comment (haus) [inline simulation-runner/src/main/scala/com/puppetlabs/gatling/node_simulations/OpsWorksLarge.scala]: Was this recording using glisan or hoyt?

**Comment by haus** [comment]
@doug-rosser It looks like all of the simulations you've checked in here are running 4.10.x agents. Was that intentional?

**Comment by haus** [comment]
I guess that makes sense since most of the perf was run against glisan.

File: simulation-runner/src/main/scala/com/puppetlabs/gatling/node_simulations/OpsWorksLarge.scala (hunk 2 above)
Comment (haus) [reply simulation-runner/src/main/scala/com/puppetlabs/gatling/node_simulations/OpsWorksLarge.scala]: Never mind, I'm remembering that most of the perf testing was against glisan.

**Comment by samwoods1** [comment]
I'm 👍

**Pull request merged**